.github/actions/setup-deps/action.yaml

```diff
@@ -45,6 +45,8 @@ inputs:
     default: 'pytest'
   scipy:
     default: 'scipy'
+  transformations:
+    default: 'transformations'
```

Comment:
I thought adding transformations here would install it as part of CI but I still see it as pypi installed https://github.com/MDAnalysis/mdanalysis/actions/runs/15578157334/job/43867072733?pr=5062#step:7:269
```
  transformations                   2025.1.1      pypi_0                   pypi    
```
What should I be doing?

Reply:
I got time this weekend, will look into it!

Reply:
Just to expand, I suspect that there is something deeper with the way in which conda recognises the package. We should check this before making the switch, otherwise we might end up with and rdkit pypi situation, except from a non-optional package.

Reply:
It might be as easy as merging a bunch of PRs https://github.com/conda-forge/transformations-feedstock/pulls ... or not ;-)

Reply:
I wouldn't say it is an orphan packaged, because PyPI is up to date and there has been a 2025 release. So I think the main issue is with the conda-forge feedstock, which we could take over?